src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,101 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int SIZE_INCREASE_INDEX = 2;
+    private Node<K, V>[] map;
+    private int threshold;
+    private int size;
+
+    private static class Node<K, V> {
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(K key, V value, Node<K, V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
+
+    public MyHashMap() {
+        this.map = new Node[INITIAL_CAPACITY];
```

Comment:
```suggestion
        map = new Node[INITIAL_CAPACITY];
```

we don`t need `this` keyword here